### PR TITLE
Merge develop into mbaas-data

### PR DESCRIPTION
**Pull request by jackycodes**
Things to consider before you submit the PR:

* [ ] Has `CHANGELOG.md` been updated?
* [ ] Are tests passing locally?
* [ ] Are the files formatted correctly?
* [ ] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

Update the mbaas-data branch with the latest develop

### CODE REVIEW

File: appcenter-crashes/appcenter-crashes.podspec [hunk 1]
```diff
@@ -20,6 +20,6 @@ Pod::Spec.new do |s|
   s.vendored_frameworks = 'AppCenterReactNativeShared/AppCenterReactNativeShared.framework'
 
   s.dependency 'AppCenterReactNativeShared'
-  s.dependency 'AppCenter/Crashes', '2.2.1-5'
+  s.dependency 'AppCenter/Crashes', '2.2.0'
```
Comment (guperrot) [inline appcenter-crashes/appcenter-crashes.podspec]: same here

File: AppCenterReactNativeShared/Products/local.podspec [hunk 2]
```diff
@@ -14,5 +14,5 @@ Pod::Spec.new do |s|
   s.platform          = :ios, '9.0'
   s.requires_arc      = true
   s.vendored_frameworks = 'AppCenterReactNativeShared/AppCenterReactNativeShared.framework'
-  s.dependency 'AppCenter/Core', '2.2.1-5'
+  s.dependency 'AppCenter/Core', '2.2.0'
```
Comment (guperrot) [inline AppCenterReactNativeShared/Products/local.podspec]: we need the prerelease, please leave that version as is.

File: appcenter-crashes/appcenter-crashes.podspec (hunk 1 above)
Comment (jackycodes) [reply appcenter-crashes/appcenter-crashes.podspec]: Changed

File: AppCenterReactNativeShared/Products/local.podspec (hunk 2 above)
Comment (jackycodes) [reply AppCenterReactNativeShared/Products/local.podspec]: Changed

**Comment by jackycodes** [comment]
@guperrot You might need to regenerate the Podfile.lock since I don't have access to the prerelease.

**Pull request merged**